examples/analytics.amp.html

```diff
@@ -218,6 +218,25 @@
 </script>
 </amp-analytics>
 
+<!-- Lotame tracking.
+  Please replace the value for "account" (12345)
+  with the correct Lotame Client ID.
+-->
+<amp-analytics type="lotame">
+<script type="application/json">
+{
+  "vars": {
+    "account": 12345
+  },
+  "extraUrlParams": {
```

Comment:
I think keeping it here is fine. But can you change the values to something meaningful? What do you expect publishers to use this for most?